mbed_targets/__init__.py

```diff
@@ -1,4 +1,24 @@
-"""Exposes the primary interfaces for the library."""
+"""mbed-targets provides an abstraction layer for boards and platforms supported by Mbed OS.
 
+This package is intended for use by developers of Mbed OS.
+
+To fetch data about private targets, an authentication token must be provided in an
```

Comment:
The user will also need to have an account on os.mbed.com and be member of a vendor team that has permissions to see the private board.

Reply:
Good point. I'll add a note about that.